php-config/Slate/People/Student.config.d/current-sections-relationship.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Slate\People;
+
+use DB;
+
+use Slate\Courses\Section;
+use Slate\Courses\SectionParticipant;
+use Slate\Term;
+
+$courseSectionsInTerm = DB::allValues(
+    'ID',
+    '
+        SELECT ID
+          FROM `%s`
+         WHERE TermID IN (%s)
+    ',
+    [
+        Section::$tableName,
+        join(',', Term::getClosest()->getMaster()->getContainedTermIDs())
```

Comment:
This logic is "all sections for this school year" but isn't "all current sections", that would be `Term::getClosest()->getConcurrentTermIDs()`

So maybe call this relationship `CurrentMasterSections` and add a real `CurrentSections` while we're at it

Reply:
Let's also add a static method `Term::getClosestConcurrentTermIDs()` that stores its array in the global `Cache` like [`ContentArea`](https://github.com/SlateFoundation/slate-cbl/blob/develop/php-classes/Slate/CBL/ContentArea.php#L71) does, and clear it at the end of `Term`'s `save()` override. This will give us quick access to the list of current TermIDs without hitting the database. Let's do `Term::getClosestMasterConcurrentTermIDs()` too

Reply:
@themightychris just wanted to confirm that you want a `Term::getClosestMasterConcurrentTermIDs()` method that returns a cached
`Term::getClosest()->getMaster()->getConcurrentTermIDs()`
or should it return what I have above?
`Term::getClosest()->getMaster()->getContainedTermIDs()`

Reply:
Oh good catch, you're right